lib/SIL/SILFunctionBuilder.cpp

```diff
@@ -35,8 +35,6 @@ SILFunction *SILFunctionBuilder::getOrCreateFunction(
   return fn;
 }
 
-
-// SWIFT_ENABLE_TENSORFLOW
```

Comment:
Isn't this still different from upstream?

Reply:
Ok, just saw your new comment.

Reply:
See https://github.com/apple/swift/pull/22280/files#r252896538.
I believe upstream (current `master`, at least) is identical now.